applications/relaycode.md

```diff
@@ -0,0 +1,188 @@
+
+Relaycode addresses the need for a more accessible and efficient extrinsics builder, as identified through:
+- Feedback from the Polkadot developer community
+- Discussions in Polkadot forums and social channels
+- Personal experience with existing tools
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Yogesh Kumar
+- 1 FTE (To be decided)
+
+### Contact
+
+- **Contact Name:** Yogesh Kumar
+- **Contact Email:** itsyogesh@outlook.com
+- **Website:** https://github.com/itsyogesh
+
+### Legal Structure
+
+- **Registered Address:** 5/12, West Patel Nagar, New Delhi, India
+- **Registered Legal Entity:** Eva Interactive Co.
+
+### Team's experience
+
+Yogesh Kumar has 10+ years of experience in fullstack development, with a focus on Blockchain development for the past 5 years. He has contributed to various projects and built products on Bitcoin, Ethereum and Polkadot ecosystem.
+
+
+Our team has not previously applied for a Web3 Foundation grant.
+
+### Team Code Repos
+
+- https://github.com/itsyogesh
+- https://github.com/itsyogesh/polkadot-hackathon-2024/blob/main/singapore/27-Relaycode/
+
+### Team LinkedIn Profiles
+
+- https://www.linkedin.com/in/itsyogesh
+
+## Development Status :open_book:
+
+Relaycode was initially developed as part of a hackathon project. The current prototype demonstrates the basic functionality of the extrinsics builder and information pane. We have implemented:
+
+- A basic UI layout using React and Tailwind CSS
+- Integration with the Dedot library for fetching pallet and function data
+- Simple encoding and decoding of extrinsic data
+
+[Hackathon Project](https://github.com/itsyogesh/polkadot-hackathon-2024/blob/main/singapore/27-Relaycode/README.md)
+
+We have received positive feedback from the hackathon judges and participants, encouraging us to further develop and refine the project.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):** 2
+- **Total Costs:** 30,000 USD
+- **DOT %:** 50%
+
+### Milestone 1 — MVP Development and Basic Deployment
+
+- **Estimated Duration:** 1 month
+- **FTE:** 2
+- **Costs:** 15,000 USD
+- **DOT %:** 50%
+
+|  Number | Deliverable                   | Specification                                                                                                                                            |
+| ------: | ----------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **0a.** | License                       | Apache 2.0                                                                                                                                               |
+| **0b.** | Documentation                 | We will provide basic documentation including: <br>- Inline code documentation<br>- A simple tutorial explaining how to use the MVP version of Relaycode |
```

Comment:
please don't use any html tags.

Reply:
Added changes in https://github.com/w3f/Grants-Program/pull/2410/commits/b4118a1b4dcae827976ee280fbf0339e9ccf7504